### PR TITLE
Use `*` and `/` args syntaxes from Python 3.9 wherever emulated

### LINKED ISSUE #698: Use `*` and `/` args syntaxes from Python 3.9 wherever emulated
Research removing:
1. usages of `KeywordOnlyArgumentCheatcodeException`
2. exception raised from `CairoStruct.__init__`

### PR DESCRIPTION
**Pull request by mkaput**
fix #698

### CODE REVIEW

File: protostar/testing/cheatcodes/reflect/cairo_struct.py [hunk 1]
```diff
@@ -1,45 +1,31 @@
...
     def __getattr__(self, name: str) -> "CairoValueType":
         if name in self._ordered_dict:
             return self._ordered_dict[name]
-        raise CairoStructException(f'"{name}" is not a member of this CairoStruct.')
+        raise KeyError(f"'{name}' is not a member of this CairoStruct.")
 
     def __setattr__(self, name: str, value: Any) -> None:
         if name != "_ordered_dict":
-            raise CairoStructException("CairoStruct is immutable.")
+            raise ValueError("CairoStruct is immutable.")
```
Comment (kasperski95) [inline protostar/testing/cheatcodes/reflect/cairo_struct.py:23]: These exceptions will be thrown with a stacktrace as an unexpected exception.

File: protostar/testing/fuzzing/strategies/integers.py [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
+    return IntegersStrategyDescriptor(min_value, max_value)
+
+
 class IntegersStrategyDescriptor(StrategyDescriptor):
     # NOTE: Keeping these names matching arguments of Hypothesis' ``integers`` strategy,
     #   so that Hypothesis' exceptions (which contain these names) make sense for our users.
-    def __init__(
-        self, min_value: Optional[int] = None, max_value: Optional[int] = None
-    ):
+    def __init__(self, min_value: Optional[int], max_value: Optional[int]):
```
Comment (kasperski95) [inline protostar/testing/fuzzing/strategies/integers.py:20]: A user will need to pass `None` explicitly. Is it intended?

File: protostar/testing/fuzzing/strategies/one_of.py [hunk 3]
```diff
@@ -1,24 +1,23 @@
...
             if not isinstance(cast(Any, strategy), StrategyDescriptor):
-                raise SearchStrategyBuildError(
-                    "Strategy 'one_of' takes only other strategies as arguments."
-                )
+                raise TypeError("one_of() takes only other strategies as arguments")
         if len(strategies) == 0:
-            raise SearchStrategyBuildError(
-                "Strategy 'one_of' takes at least one argument."
-            )
+            raise TypeError("one_of() takes at least 1 positional argument")
```
Comment (kasperski95) [inline protostar/testing/fuzzing/strategies/one_of.py:19]: Up

File: protostar/testing/fuzzing/strategies/integers.py (hunk 2 above)
Comment (mkaput) [reply protostar/testing/fuzzing/strategies/integers.py:20]: Init is not user facing function now, there is a wrapper named `felts` (the same as strategy, it's important) that has defaults and proper interface

File: protostar/testing/cheatcodes/reflect/cairo_struct.py (hunk 1 above)
Comment (mkaput) [reply protostar/testing/cheatcodes/reflect/cairo_struct.py:23]: Discussed this in office.

File: protostar/testing/fuzzing/strategies/one_of.py (hunk 3 above)
Comment (mkaput) [reply protostar/testing/fuzzing/strategies/one_of.py:19]: Discussed this in office.

**Pull request closed without merging**